src/InputWithLabel.js

```diff
@@ -0,0 +1,22 @@
+import React ,{useState,useRef, useEffect } from 'react'
+
+function InputWithLabel(props) {
+ const inputRef = useRef();
+ useEffect(()=>{
+  if (inputRef.current) {inputRef.current.focus()};
+ })
+ return (
+  <div>
+   <label htmlFor="TodoTitle">{props.children}</label>
+   <input 
+   ref={inputRef} 
+   onChange={props.handleTitleChange} 
+   value={props.todoTitle} type="text"
+    name="title" id="todoTitle">
+</input>
```

Comment:
input closing tag not necessary since you aren't putting anything between tags. Can use a self closing tag